app/models/apple/publisher.rb

```diff
@@ -264,7 +264,14 @@ def mark_delivery_files_uploaded!(eps)
         pdfs = eps.map(&:podcast_delivery_files).flatten
         ::Apple::PodcastDeliveryFile.mark_uploaded(api, pdfs)
 
+        # link the podcast container with the audio to the episode
         res = Apple::Episode.update_audio_container_reference(api, eps)
+        # update the feeder episode to indicate that delivery is no longer needed
+        eps.each do |ep|
+          Rails.logger.info("Marking episode as no longer needing delivery", {episode_id: ep.feeder_episode.id})
+          ep.feeder_episode.apple_has_delivery!
```

Comment:
Could this be a race condition?

1. Media version A comes in, fires a publish job
2. We get partway through the job, media version B comes in, fires another publish job
3. We get here, mark delivered = true

I'm not sure that's actually a problem - probing for new media versions via #858 is the real fix for the why-did-the-user-swap-media-files-so-immediately conditions.

And this explicit control to force re-uploading is handy in any case.